checker/src/main/java/org/checkerframework/checker/nullness/NullnessNoInitSubchecker.java

```diff
@@ -20,6 +20,18 @@
  *
  * <p>The {@link NullnessChecker} uses this checker as the target (see {@link
  * InitializationChecker#getTargetCheckerClass()}) for its initialization type system.
+ *
+ * <p>You can use the following {@link SuppressWarnings} prefixes with this checker:
+ *
+ * <ul>
+ *   <li>{@code @SuppressWarnings("nullness")} suppresses warnings for nullness, initialization, and
+ *       keyfor annotations
+ *   <li>{@code @SuppressWarnings("nullnesskeyfor")} suppresses warnings for nullness and keyfor
```

Comment:
For each of the options, would it be clearer to also list what is not suppressed, to make that also explicit?